### DISCUSSION TITLE
--purge-interval usage

### DISCUSSION THREAD

**Discussion by grafpaper1987** [ep1cman/unifi-protect-backup#97] [Q&A] [answered]
Is the input for this argument hours? Days? Can someone show me how I should be using this?

Thanks!

**Comment by ep1cman** [comment]
As per the command help this argument is:

```
How frequently to check for file to purge.
                                  
                                  NOTE: Can create a lot of API calls, so be careful if your cloud
                                  provider charges you per api call  [default: 1d]
```

**Reply by grafpaper1987** [reply]
so if I wanted to purge after 60 days, the input would be: 

--purge-interval '60d' 

or

--purge-interval '60'

The help/documentation for this command isn't as clear/explicit as the help is for the other commands. Thanks.

**Reply by ep1cman** [reply, marked as answer]
'60d', if you wanted 12 hours it would be '12h' etc

**Reply by grafpaper1987** [reply]
Understood. Thank you! This tool is great by the way.

**Comment by bionic80** [comment]
Do you NEED to use the rclone config to adjust the purge time?  I'd like to go beyond the original 7d retention.

**Reply by ep1cman** [reply]
both the purge interval and retention period are set via command line arguments or environment variables. Please see the command help text in the readme file
